specification/synapse/data-plane/Microsoft.Synapse/preview/2022-03-01-preview/runNotebook.json

```
@@ -0,0 +1,562 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-03-01-preview",
+    "title": "ArtifactsClient"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "type": "oauth2",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      },
+      "description": "Azure Active Directory OAuth2 Flow"
+    }
+  },
```

Recommend to add `tags` at the top level to specify a description for the "RunNotebook" tag.

---

@mikekistler could you please provide an example about how to add top-level "tags"? I cannot find one by myself

---

Here's the section of the OpenAPI v2 spec that describes "tags", and I've included a suggestion.
```suggestion
  },
  "tags": [
    "name": "RunNotebook",
    "description": "Start, query, and cancel Notebook execution"
  ],
```

---

The grammar is not correct and it causes CI failure: https://github.com/Azure/azure-rest-api-specs/pull/16858/checks?check_run_id=5329657639